### PR TITLE
Update Attribution

### PR DESCRIPTION
**Pull request by shahar4**
### WHAT?

This PR adds attribution to the plonky2 team.

### CODE REVIEW

**Review by arose0 — APPROVED** [review]
LGTM!

File: LICENSE-MIT [hunk 1]
```diff
@@ -1,3 +1,6 @@
+This software includes components from third parties. For a full list of these components and their licenses,
+see the THIRD PARTY NOTICES file.
+
 MIT License
 
 Copyright (c) 2019 Matter Labs
```
Comment (paulmillr) [inline LICENSE-MIT:6]: You should add plonky2 copyright here instead of adding "notice" to the header.

**Review by paulmillr — APPROVED** [review]

File: LICENSE-MIT (hunk 1 above)
Comment (paulmillr) [inline LICENSE-MIT:6]: You should add plonky2 copyright here instead of adding "notice" to the header.

**Review by paulmillr — CHANGES_REQUESTED** [review]

File: LICENSE-MIT (hunk 1 above)
Comment (arose0) [reply LICENSE-MIT:6]: @paulmillr we discussed the right pattern to follow here, and ultimately this seems to be one that can keep a relatively clean top-level license file, and add all of the appropriate attribution for third party development. 

This is a pretty established approach, as best we understand it. 

e.g. https://github.com/microsoft/vscode/blob/main/ThirdPartyNotices.txt

**Review by githubdoramon — APPROVED** [review]

File: LICENSE-MIT (hunk 1 above)
Comment (paulmillr) [reply LICENSE-MIT:6]: The thirdpartynotices you've linked is a list of all dependencies. You don't depend on plonky2, it's not your third party lib.

Comment (paulmillr) [reply LICENSE-MIT:6]: So you have a derived work, not a third party dependency.

Comment (paulmillr) [reply LICENSE-MIT:6]: https://opensource.stackexchange.com/questions/4506/what-license-do-i-apply-to-my-software-if-parts-of-its-code-is-derived-from-mit

**Review by anton-matterlabs — APPROVED** [review]

File: LICENSE-MIT (hunk 1 above)
Comment (arose0) [reply LICENSE-MIT:6]: Thanks, we've confirmed with legal counsel that our approach is valid.

Comment (paulmillr) [reply LICENSE-MIT:6]: awesome

**Pull request merged**